### ISSUE TITLE
feat: warn against split apk from storage

### ISSUE THREAD

**Issue by Martinligabue** [ReVanced/revanced-manager#1342]
### Feature description

The manager should be able to recognize split apk, apart from the version.

Another way would be to explain the users where to download the apk that's not split

### Motivation

I might download the exact version of YouTube, and the patching will fail because it's split

### Additional context

I had no clue these things existed, and even if the manager was saying that the version was good, I couldn't patch the youtube app, so I opened an issue here: https://github.com/ReVanced/revanced-patches/issues/3105

They said I have to report here

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

**Comment by oSumAtrIX** [comment]
Not sure how much it is the task of ReVanced Manager to do that.

**Comment by Ushie** [comment]
> Not sure how much it is the task of ReVanced Manager to do that.

It is equivalent to the warning that's shown when selecting an installed app

**Comment by oSumAtrIX** [comment]
The warning is also not correct. An installed application can be patched, only splits can not be.

**Comment by Ushie** [comment]
The warning isn't shown if the installed app isn't split

**Comment by oSumAtrIX** [comment]
Alright

**Comment by Martinligabue** [comment]
I did not get any warning

**Comment by Ushie** [comment]
Did you install it on Aurora store and then select it? I might've misinterpreted the issue if so

**Comment by Martinligabue** [comment]
yes, I just chose the manual download to download a specific version instead of the latest.

but as they answered in the other issue, that results in a split apk, whatever that means, so it's not a viable option to install a YouTube apk that this patcher likes. (The only way is to search for it online from third parties who repackage it as a single apk, afai understand?)

**Comment by sigma2017** [comment]
The latest version requires v18.38.44 which is available for download only as xapk and manager can't use this type of file.
Waiting for a newer version...

**Comment by Ushie** [comment]
It's available as .apk on APKMirror

**Comment by mitian233** [comment]
Google has now stopped providing fat APKs for sideloading on the Play Store. Consequently, most third-party websites that used to provide application packages have also stopped providing .apk, instead providing split APKs, which cannot be directly patched by ReVanced Manager.

**Comment by coderZalmy** [comment]
I was able to get a proper apk from Aurora. But it still only worked when I patched the one from apkmirror
